libraries/botbuilder-dialogs-adaptive-testing/src/httpRequestMocks/httpResponseMockContent.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * @module botbuilder-dialogs-adaptive-testing
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { HttpResponseMock, ResponseContentType } from './httpResponseMock';
+
+export class HttpResponseMockContent {
+    private readonly _contentType: ResponseContentType;
+    private readonly _content: string | Record<string, unknown>;
+
+    public constructor(httpResponseMock?: HttpResponseMock) {
+        if (!httpResponseMock) {
```

Comment:
This could be a touch more compact:

```typescript
public constructor(httpResponseMock?: HttpResponseMock) {
  this._contentType = httpResponseMock?.contentType ?? ResponseContentType.String;
  this._content = httpResponseMock?.content ?? '';
}
```